pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>token-macro</artifactId>
       <version>1.10</version>
     </dependency>
+    <dependency>
+      <groupId>com.github.docker-java</groupId>
+      <artifactId>docker-java</artifactId>
+      <version>1.3.0</version>
```

Comment:
IOW if docker-plugin depends on docker-commons rather than docker-java then that is "ok".

better is if everyone depends on `docker-java-api` (or whatever you call the wrapper plugin)